public/index.html

```diff
@@ -26,7 +28,7 @@
 
 <body>
   <noscript>You need to enable JavaScript to run this app.</noscript>
-  <div id="root"></div>
+  <div id="root" style="min-height: 100vh"></div>
```

Comment:
@Alexandria It's not good practice to add styles this way, if this is necessary than you should use an imported css file to add this style.